bsoid_app/extract_features.py

```diff
@@ -16,15 +16,18 @@
 from bsoid_app.bsoid_utilities.load_workspace import load_feats, load_embeddings
 from bsoid_app.config import *
 
+framerate = os.environ.get('framerate', '')
```

Comment:
You don't need these actually. You can pass them as input variables to `extract` in `bsoid_app.py` when you initialize a la:

```
extract_features.extract(
        working_dir, prefix, processed_input_data, framerate
    )
```